components/proposal-merge.tsx

```diff
@@ -54,20 +58,32 @@ export default function ProposalMerge({
       proposals: [defaultAmount],
     });
 
-  const { t } = useTranslation(["common", "proposal"]);
-
-  useEffect(() => {
-    getDistributedAmounts();
-  }, [proposal]);
+  const { activeNetwork } = useNetwork();
 
+  
   async function getDistributedAmounts() {
     if (!proposal?.details) return;
+    
+    const distributions = calculateDistributedAmounts(activeNetwork?.treasury,
+                                                      activeNetwork?.mergeCreatorFeeShare,
```

Comment:
lint will kill you here; You can escape it by breaking line after the `=` sign on 67 and then indent each line with 2 spaces:

```ts
const distributions =
  calculateDistributedAmounts(activeNetwork?.treasury,
    activeNetwork?.mergeCreatorFeeShare, // use 4 spaces here
    /* ... */);
```